<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>海报模板预览</string>
    </property>
    <widget class="QLabel" name="model">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string>加载模板失败</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>头像预览</string>
    </property>
    <widget class="QLabel" name="head">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>加载头像失败</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>二维码预览</string>
    </property>
    <widget class="QLabel" name="erweima">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>加载二维码失败</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>裁剪</string>
    </property>
    <widget class="QLabel" name="head_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>加载裁剪头像失败</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>提取作者以及文章标题</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>请选择excl：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>生成控制</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>裁剪图片</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>处理图片</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>生成个数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>生成进度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>测试替换图片</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>统一分辨率</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>图片裁剪</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>字体选择作者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_font_author">
       <property name="text">
        <string>未选择字体</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>字体选择文章</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_font_title">
       <property name="text">
        <string>未选择字体</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图片定位设置</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>checkBox_4</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
